packages/auth/amplify_auth_cognito_dart/lib/src/jwt/src/cognito.dart

```diff
@@ -89,22 +89,6 @@ extension CognitoAccessToken on JsonWebToken {
 /// - [OpenID Connect Core: ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken)
 /// - [OpenID Connect Core: Standard Claims](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims)
 extension CognitoIdToken on JsonWebToken {
-  /// The `nonce` claim comes from a parameter of the same name that you can add
-  /// to requests to your OAuth 2.0 `authorize` endpoint. When you add the
-  /// parameter, the `nonce` claim is included in the ID token that Amazon
-  /// Cognito issues, and you can use it to guard against replay attacks.
-  ///
-  /// If you do not provide a nonce value in your request, Amazon Cognito
-  /// automatically generates and validates a nonce when you authenticate
-  /// through a third-party identity provider, then adds it as a nonce claim to
-  /// the ID token.
-  ///
-  /// The implementation of the nonce claim in Amazon Cognito is based on OIDC
-  /// standards.
-  ///
-  /// [Reference](https://docs.aws.amazon.com/cognito/latest/developerguide/amazon-cognito-user-pools-using-the-id-token.html)
-  String? get nonce => claims.customClaims['nonce'] as String?;
```

Comment:
Should this be deprecated instead of removed?

Reply:
Hmm yeah, it should. Good catch.

Reply:
I actually just put it back. Customers can use it if they want.